cgi/org.pl

```diff
@@ -384,6 +386,30 @@
 		}
 	}
 
+	elsif ($type eq 'admin_status') {
+		if (is_user_in_org_group($org_ref, $User_id, "admins") or $admin or $User{pro_moderator}) {
+			foreach my $user_id (sort keys %{$org_ref->{members}}) {
+				my $checkbox_name = "admin_status_hidden_" . $user_id;
+				my $is_admin = single_param($checkbox_name) ? 1 : 0;
+				$log->debug("checkbox name", {checkss => $checkbox_name, statuss => $is_admin}) if $log->is_debug();
+
+				if ($is_admin) {
+					add_user_to_org($org_ref, $user_id, ["admins"]);
+					unless (grep {$_ == $user_id} @admin_status) {
+						push @admin_status, $user_id;
+
+					}
+				}
+				else {
+					remove_user_from_org($org_ref, $user_id, ["admins"]);
+					@admin_status = grep {$_ != $user_id} @admin_status;
+				}
+				store_org($org_ref);
```

Comment:
store_org() should be outside the for loop, otherwise we will save the org as many times as there are users